src/scenes/home/Task.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Image, Text, TouchableOpacity, View } from 'react-native'
+import { NavigationRoute, NavigationScreenProp } from 'react-navigation'
+import { connect } from 'react-redux'
+
+import { Actions } from 'src/actions'
+import { Route } from 'src/helpers/Route'
+import { styles } from 'src/scenes/home/Task.styles'
+
+import iconActive from 'src/assets/iconCheckboxActive.png'
+import iconInactive from 'src/assets/iconCheckboxInactive.png'
+import { ActionCreator } from 'redux'
+import { toggleDoneAction } from 'src/actions/types'
+
+export interface Props {
+  url: string
+  title: string
+  description: string
+  completed: boolean
+  navigation: NavigationScreenProp<NavigationRoute>
+  toggle: (id: string) => void
+}
+
+function layoutTask(props: Props) {
```

Comment:
change this for
`export const LayoutTask:React.FunctionComponent<Props> = () => {`

1 - we use always arrow functions.
3 - just check this in every file